foursight_core/react/api/misc_utils.py

```diff
@@ -37,6 +38,14 @@ def get_request_body(request: Request) -> dict:
     return json.loads(request.raw_body.decode())
 
 
+def get_base_url(url: str) -> str:
```

Comment:
Added doc string. For my own info, how might ``.netloc`` contain credentials?

Reply:
Oooh, you URL like ... http://username:password@example.com ... Yes? Will take care of this. I don't want any such info in the URL - shouldn't come up anyways, but you're right should handle and/or document.

Reply:
I think I just need to change the ``url.netloc`` to ``url.hostname``, that seems to do it.

Reply:
Yeah, I think that's right. 

You might optionally want to add something like this (untested):
```
port = '' if (url.port == 80 and url.scheme == 'http') or (url.port == 443 and url.scheme == 'https') else url.port
```
and then use port instead of url.port when assembling the URL.

Either way, you can resolve this comment if you want.

Reply:
Oh yeah, good idea, thanks.